source/IO/dataset_import/AIRRImport.py

```diff
@@ -1,18 +1,75 @@
+from source.data_model.receptor.receptor_sequence.SequenceFrameType import SequenceFrameType
+from source.data_model.repertoire.Repertoire import Repertoire
 from source.util.ImportHelper import ImportHelper
 
 
 class AIRRImport(DataImport):
     """
-    Imports the data from an AIRR-formatted .tsv files into a RepertoireDataset.
+    Imports data in AIRR format into a Repertoire-, or SequenceDataset.
+    RepertoireDatasets should be used when making predictions per repertoire, such as predicting a disease state.
+    SequenceDatasets should be used when predicting values for for unpaired (single-chain) immune receptors, like
+    antigen specificity.
 
     AIRR rearrangement schema can be found here: https://docs.airr-community.org/en/stable/datarep/rearrangements.html
 
+
+    Arguments:
+        path (str): Required parameter. This is the path to a directory with AIRR files to import.
+
+        is_repertoire (bool): If True, this imports a RepertoireDataset. If False, it imports a SequenceDataset.
+            By default, is_repertoire is set to True.
+
+        metadata_file (str): Required for RepertoireDatasets. This parameter specifies the path to the metadata file.
+            This is a csv file with columns filename, subject_id and arbitrary other columns which can be used as labels in instructions.
+            Only the AIRR files included under the column 'filename' are imported into the RepertoireDataset.
+            For setting SequenceDataset metadata, metadata_file is ignored, see metadata_column_mapping instead.
+
+        import_productive (bool): Whether productive sequences (with value 'T' in column productive) should be included
+            in the imported sequences. By default, import_productive is True.
+
+        import_with_stop_codon (bool): Whether sequences with stop codons (with value 'T' in column stop_codon) should
+            be included in the imported sequences. This only applies if column stop_codon is present. By default,
+            import_with_stop_codon is False.
+
+        import_out_of_frame (bool): Whether out of frame sequences (with value 'F' in column vj_in_frame) should
+            be included in the imported sequences. This only applies if column vj_in_frame is present. By default,
+            import_out_of_frame is False.
+
+        region_type (str): Which part of the sequence to import. By default, this value is set to IMGT_CDR3. This means the
+            first and last amino acids are removed from the CDR3 sequence, as AIRR uses the IMGT junction. Specifying
+            any other value will result in no trimming of the imported sequences.
+            Valid values for region_type are the names of the :py:obj:`~source.data_model.receptor.RegionType.RegionType` enum.
+
+        column_mapping (dict): A mapping from AIRR column names to immuneML's internal data representation.
+            For AIRR, this is by default set to:
+                junction: sequences
```

Comment:
because the junction field is required and will always be present in AIRR files, while the CDR3 field is optional. If we rely on the junction field it will always work, and at worst require this one extra operation of removing leading/trailing aa's. If we rely on CDR3, the importer will return errors for many(!) AIRR files and the user manually has to overwrite the column mapping to overcome this.